projects/demo/src/pages/phone/phone-doc.component.ts

```diff
@@ -0,0 +1,44 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {DocExamplePrimaryTab} from '@demo/constants';
+import {maskitoPhoneOptionsGenerator} from '@maskito/phone';
+import {TuiDocExample} from '@taiga-ui/addon-doc';
+import {CountryCode, getCountries} from 'libphonenumber-js/core';
+import metadata from 'libphonenumber-js/min/metadata';
+
+type GeneratorOptions = Required<Parameters<typeof maskitoPhoneOptionsGenerator>[0]>;
+
+@Component({
+    selector: 'phone-doc',
+    templateUrl: './phone-doc.template.html',
+    styleUrls: ['./phone-doc.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PhoneDocComponent implements GeneratorOptions {
```

Comment:
what about example

```ts
async ngOnInit() {
    this.mask = maskitoPhoneOptionsGenerator({
      countryIsoCode: 'KZ',
      metadata: await import('libphonenumber-js/min/metadata')
          .then(({ default }) => default),
    });
```

Reply:
An example with lazy metadata will be added to a separate PR